docs/appendix.rst

```diff
@@ -143,6 +143,25 @@ This documentation uses a few special terms to refer to Python types:
       backslash-escaping any double quotes in INSTANCENAME, and putting it into
       double quotes.
 
+   default connection
+      A connection definition in the :term:`connections file` that is used
+      by pywbemcli as the :term:`current connection` if pywbemcli is started without
+      any connection definition (no :ref:`--server general option`,
+      :ref:`--mock-server general option`, or :ref:`--name general option`) and
+      the default connection is defined.
+      A connection definition in the :term:`connections file` becomes the
+      default connection on pywbemcli startup if it is defined using the
+      :ref:`connection select command` and the ``--default option`` is used.
+
+   current connection
+      The connection definition in pywbemcli that is currently active; it is the
+      target connection for pywbemcli commands.  The current
+      server is created on pywbemcli startup with the following options
```

Comment:
Change "current server" to "current connection".

Reply:
DONE